ui/components/app/asset-list/detetcted-tokens-link/index.scss

```diff
@@ -0,0 +1,5 @@
+.detected-tokens-link {
+  & &__link {
+    @include H6;
+  }
+}
```

Comment:
@georgewrmarshall we might consider adding a "typographyVariant" or some such prop to the Button component if this becomes a standard use case.  (not a change request or NIT @NiranjanaBinoy , just calling attention to possible design system component updates :))

Reply:
Right. The other option is to use `type="inline"` which inherits the font-size from it's parent. I see your point though if the component isn't inside a parent with a font-size how does the developer match the designs.